src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-size: 30px;
+  padding: $square-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-spacing;
+  width: $calendar-width;
```

Comment:
The variable `$calendar-width` is used here. Verify that this variable is defined in your SCSS files.